package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-template-node",
-  "version": "0.0.0",
+  "version": "0.1.0",
```

Comment:
Love bikeshadding :)   This is a template - I don't think we should ever increase this version number - not much point in it.  On the other hand, the services based on this code should have 0.1.0 as the first version, thus setting it here so people don't have to change it right away.